build/NuSpecs/Microsoft.ProjectReunion.Foundation.props

```diff
@@ -0,0 +1,6 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- <Import Project="$(MSBuildThisFileDirectory)\MicrosoftProjectReunionVersion.props"/> -->
```

Comment:
>MicrosoftProjectReunionVersion.props [](start = 52, length = 36)

I assume these are sort of like "TODOs"?

Reply:
Yea, right now this props file assumes the AppX is inside the thin nuget package with it, but it is not.

We intend for the AppX to be inside another package, the metapackage.  So we'll have to figure out how this will work in the future.